unit-tests/dds/test-librs-device-properties.py

```diff
@@ -0,0 +1,124 @@
+    #
+    test.start( "Test D435i" )
+    try:
+        remote.run( 'instance = broadcast_device( d435i, d435i.device_info )', timeout=5 )
+        n_devs = 0
+        for dev in wait_for_devices():
+            n_devs += 1
+        test.check_equal( n_devs, 1 )
+        test.check_equal( dev.get_info( rs.camera_info.name ), d435i.device_info.name )
+        test.check_equal( dev.get_info( rs.camera_info.serial_number ), d435i.device_info.serial )
+        test.check_equal( dev.get_info( rs.camera_info.physical_port ), d435i.device_info.topic_root )
+        sensors = {sensor.get_info( rs.camera_info.name ) : sensor for sensor in dev.query_sensors()}
+        test.check_equal( len(sensors), 3 )
+        if test.check( 'Stereo Module' in sensors ):
+            sensor = sensors.get('Stereo Module')
+            test.check_equal( len(sensor.get_stream_profiles()), len(d435i.depth_stream_profiles())+2*len(d435i.ir_stream_profiles()) )
+        if test.check( 'RGB Camera' in sensors ):
+            sensor = sensors['RGB Camera']
+            test.check_equal( len(sensor.get_stream_profiles()), len(d435i.color_stream_profiles()) )
+        if test.check( 'Motion Module' in sensors ):
+            sensor = sensors['Motion Module']
+            test.check_equal( len(sensor.get_stream_profiles()), len(d435i.accel_stream_profiles())+len(d435i.gyro_stream_profiles()) )
+        remote.run( 'close_server( instance )', timeout=5 )
+    except:
+        test.unexpected_exception()
+    dev = None
+    test.finish()
+    #
+    #############################################################################################
+    #
+    test.start( "Test D405" )
+    try:
+        remote.run( 'instance = broadcast_device( d405, d405.device_info )', timeout=5 )
+        n_devs = 0
+        for dev in wait_for_devices():
+            n_devs += 1
+        test.check_equal( n_devs, 1 )
+        test.check_equal( dev.get_info( rs.camera_info.name ), d405.device_info.name )
+        test.check_equal( dev.get_info( rs.camera_info.serial_number ), d405.device_info.serial )
+        test.check_equal( dev.get_info( rs.camera_info.physical_port ), d405.device_info.topic_root )
+        sensors = {sensor.get_info( rs.camera_info.name ) : sensor for sensor in dev.query_sensors()}
+        test.check_equal( len(sensors), 1 )
+        if test.check( 'Stereo Module' in sensors ):
+            sensor = sensors.get('Stereo Module')
+            test.check_equal( len(sensor.get_stream_profiles()),
+                              len(d405.depth_stream_profiles())+2*len(d405.ir_stream_profiles())+len(d405.color_stream_profiles())+len(d405.colored_infrared_stream_profiles()) )
+        remote.run( 'close_server( instance )', timeout=5 )
+    except:
+        test.unexpected_exception()
+    dev = None
+    test.finish()
+    #
+    #############################################################################################
+    #
+    test.start( "Test D455" )
```

Comment:
Why do we need D455 test? is it different then D435i?

Reply:
As we discussed, I added D405 so I can test multiple **different** devices. But since I had the functions that print the info ready it did not take long to add another model so I did...